build-logic/build-platform/build.gradle.kts

```diff
@@ -6,7 +6,7 @@ description = "Provides a platform that constrains versions of external dependen
 
 // Here you should declare versions which should be shared by the different modules of buildSrc itself
 val javaParserVersion = "3.18.0"
-val groovyVersion = "3.0.9"
+val groovyVersion = "3.0.10"
```

Comment:
Is this Groovy version update intentional? It seems like `master` is still on `3.0.9`.

Reply:
Yes. Someone did update to 3.0.10 on `master`, but not everywhere. This brings everything to 3.0.10.